docs/team/app-access.md

```diff
@@ -0,0 +1,21 @@
+title: Sematext App Access
+description: App Sharing is restricted only to a particular app. Nothing besides the shared app is accessible to the invited user for example, dashboards are at the account level can can thus be shared only through Account Sharing.
+
+App Access is **restricted only to a particular App**. Nothing besides the 
+shared App is accessible to the invited user. 
```

Comment:
Maybe Alert rules are?  Maybe only if you are ADMIN?
How about Events?

Reply:
I don't know. I have to check.

Reply:
I've tested it now and nothing besides the App is visible, neither Alerts or Events.